<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coinfilterDialog</class>
 <widget class="QDialog" name="coinfilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/icon/c_black.png</normaloff>:/new/icon/c_black.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="change_1h">
       <property name="text">
        <string>1h change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_1h_from">
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>-----&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_1h_to">
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="3" rowspan="2" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="change_24h">
       <property name="text">
        <string>24h change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_24h_from">
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>---&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_24h_to">
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="change_7d">
       <property name="text">
        <string>7d change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_7d_from">
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>-----&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="change_7d_to">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-50.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QCheckBox" name="pricechange">
       <property name="text">
        <string>Price change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pricechange_from">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>of 100dollars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="pricechange_to">
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="use_last_volume">
       <property name="text">
        <string>Percent of last volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="last_volume_percent">
       <property name="maximum">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QCheckBox" name="volume_greater">
     <property name="text">
      <string>Last volume to be less than new volume</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="show_only_blacklisted">
     <property name="text">
      <string>Show blacklisted pairs</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Volum min</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QDoubleSpinBox" name="volum_min">
     <property name="maximum">
      <double>1000000000.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>coinfilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>coinfilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
